cmd/minishift/cmd/profile/profile.go

```diff
@@ -19,13 +19,15 @@ package profile
 import (
 	"strings"
 
+	cmdUtil "github.com/minishift/minishift/cmd/minishift/cmd/util"
 	"github.com/minishift/minishift/pkg/util/os/atexit"
 	"github.com/spf13/cobra"
 )
 
 const (
 	emptyProfileMessage  = "A profile name must be provided. Run `minishift profile list` for a list of existing profiles."
 	extraArgumentMessage = "You have provided more arguments than required. You must provide a single profile name."
+	invalidNameMessage   = "Profile names must consist of alphanumeric characters only."
```

Comment:
`Profile names` or `Profile name` ? as we are printing the error for one profile name

```
$ minishift profile set *hello
Profile names must consist of alphanumeric characters only.
```
@robin-owen WDYT?

Reply:
@minishift/minishift-dev any comments?

Reply:
both works fine in this context IMO. using `names` generalises it that all profiles should be alpha numeric and `name` would mean the profile name supplied to the command. both are correct statements. if we want to use `name` then ` The profile name must consist of alphanumeric characters only`